missingNumbers.js

```diff
@@ -0,0 +1,20 @@
+function findMissingNumbers(inputArray) {
+    var missingNumbers = [];
+    if (!Array.isArray(inputArray)) {
+        throw new Error('Invalid Input');
+    }
+    if (inputArray.length <= 1) {
+        throw new Error('array length too small');
+    }
+    inputArray.sort((a, b) => a - b)
+    for (var i = 1; i < inputArray.length; i++) {
+        var valDifference = inputArray[i] - inputArray[i - 1];
+        if (valDifference > 1) {            
```

Comment:
I, by the way, think we don't need this `if condition` the for loop initialization and condition cater for it